apps/ergw/src/ergw_compat.erl

```diff
@@ -893,12 +893,23 @@ gtp_proxy_ds_translate_option(Opt, Value) ->
 ]).
 
 gtp_path_translate_options(Values) ->
-    #{idle_echo := IdleEcho, idle_timeout := IdleTimeout,
+    #{t3 := T3, n3 := N3,
+      echo := BusyEcho,
+      icmp_error_handling := ICMP,
+      idle_echo := IdleEcho, idle_timeout := IdleTimeout,
       down_echo := DownEcho, down_timeout := DownTimeout} =
-	C0 = translate_options(fun gtp_path_translate_option/2, Values, ?PathDefaults, map),
-    C1 = maps:without([idle_echo, idle_timeout, down_echo, down_timeout], C0),
-    C1#{idle => #{echo => IdleEcho, timeout => IdleTimeout},
-	down => #{echo => DownEcho, timeout => DownTimeout}}.
+	translate_options(fun gtp_path_translate_option/2, Values, ?PathDefaults, map),
+
+    Evs0 = #{icmp_error => warning, echo_timeout => critical},
```

Comment:
Isn't QA using a value of info for the icmp_error?

Reply:
This is used only for the conversion of old (v2.8.x) configs. The old config does not have a value for this and this mirrors the defaults use in ergw_config.